src/Exchange.ts

```diff
@@ -15,14 +14,9 @@ export enum OrderDirection {
     TOKEN_SELL /** Sell order: orderDirection is 1 in contract */
 }
 
-export type ISellOrder = IGenericOrder;
-export interface IBuyOrder extends IGenericOrder {
-    bnEthAmount: BigNumber /** Buy order bnAmount in ETH */;
-}
-
 export interface IOrderBook {
-    buyOrders: IBuyOrder[];
-    sellOrders: ISellOrder[];
+    buyOrders: IOrder[];
```

Comment:
Interface change, breaking change. We should start to get used to list these in release notes. 
I suggest to always list interface changes in PR description so we can easily put together the release notes.